src/app/router.tsx

```
@@ -4,6 +4,11 @@ import { AuthGuard } from "@/features/auth/components/AuthGuard";
 import { MainLayout } from "@/shared/components/layout/MainLayout";
 import { OverviewPage } from "@/features/overview/pages/OverviewPage";
 import { LoginPage } from "@/features/auth/pages/LoginPage";
+import OrdersPage from "@/features/orders/pages/OrdersPage";
+import ProductsPage from "@/products-page";
+import MarketplaceAccountsPage from "@/marketplace-accounts-updated";
+import ProductsPage from "@/products-page";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ProductsPage'. Consider to delete it or rename it.

'ProductsPage' is defined here:


(lint/suspicious/noRedeclare)